src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompositeTagHelperDescriptorResolver.cs

```diff
@@ -10,22 +10,18 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Internal
     public class CompositeTagHelperDescriptorResolver : ITagHelperDescriptorResolver
```

Comment:
The `ITagHelperDescriptorResolver` type here is going to be removed. Let's delete this and bring it back in Razour.